packages/gestalt/src/Flyout.test.js

```diff
@@ -1,25 +0,0 @@
-// @flow
```

Comment:
These tests are being deleted as they are duplicated in the `Flyout.jsdom.test.js` file and actually throw a flow error now for the `anchor` prop